src/main/java/ch/hevs/cloudio/endpoint/GenericJacksonMessageFormat.java

```diff
@@ -329,6 +334,14 @@ public byte[] serializeJobsLineOutput(JobsLineOutput jobsLineOutput) {
         return outputStream.toByteArray();
     }
 
+    @Override
+    public CloudioFactoryNodes deserializeNodes(byte[] data) throws Exception
```

Comment:
I would not add this here, it has nothing to do with the format of the messages. How about a CloudioFactoryFormat interface and an implementation for JSON.